src/VisualStudio/Core/Def/SolutionEventMonitor.cs

```diff
@@ -56,24 +56,24 @@ private void ContextChangedWorker(UIContextChangedEventArgs e, string operation)
         {
             if (_notificationService != null)
             {
-                var globalOperation = _operations[operation];
+                TryCancelPendingNotification(operation);
```

Comment:
I usually like

rather than if () { .. do something there }, if (!x) { return} .. do something here

since it reduces indentation - make execution path leaner.

but, I guess that is only my preference.